CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 :wrench: **Maintenance**
 
 - Install NHS.UK frontend v6.2.0
+- Added SonarQube analysis to pipeline
```

Comment:
Please remove:
1. v4.8.6 has already been released so this would require a new 'unreleased' section in the changelog
2. It has been decided that EQF items such as this will not be documented in the changelog.